applications/nrf_desktop/configuration/nrf52840dk_nrf52840/images/mcuboot/prj_mcuboot_smp.conf

```diff
@@ -33,9 +33,9 @@ CONFIG_NCS_BOOT_BANNER=n
 # Use minimal C library instead of the Picolib
 CONFIG_MINIMAL_LIBC=y
 
-# Activate Link Time Optimization (LTO)
-CONFIG_LTO=y
-CONFIG_ISR_TABLES_LOCAL_DECLARATION=y
+# Disable Link Time Optimization (LTO). LTO causes application boot failures.
+CONFIG_LTO=n
+CONFIG_ISR_TABLES_LOCAL_DECLARATION=n
 
```

Comment:
```suggestion
```
You are setting what is already set

Reply:
I am aware. I would like to do it explicitly to make users aware of the issues.